packages/form-layout/src/layouts/auto-responsive-layout.js

```diff
@@ -130,7 +130,8 @@ export class AutoResponsiveLayout extends AbstractLayout {
       host.style.removeProperty('--_column-width');
     }
 
-    host.style.setProperty('--_max-columns', Math.min(props.maxColumns, maxColumns));
+    host.style.setProperty('--_min-columns', Math.max(props.minColumns, 1));
+    host.style.setProperty('--_max-columns', Math.min(Math.max(props.minColumns, props.maxColumns, 1), maxColumns));
```

Comment:
If we want to introduce validation, it would be better handled in the property's observer, where the value can be reverted. However, it seems that this pattern isn't commonly used in our repo, so I'd suggest simply letting the application fail if an invalid minColumns is set. 

```suggestion
    host.style.setProperty('--_min-columns', props.minColumns);
    host.style.setProperty('--_max-columns', Math.min(Math.max(props.minColumns, props.maxColumns), maxColumns));
```